catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/ListApplication.java

```diff
@@ -108,7 +110,8 @@ public void init() {
 
           List<Part> parts = new ArrayList<>(request.raw().getParts());
 
-          AttachmentInfo attachmentInfo = parseAttachments(parts);
+          Pair<AttachmentInfo, Metacard> attachmentInfo =
+              catalogService.parseAttachments(partsToInfo(parts), null);
```

Comment:
❓ Why do we go through all the work of converting to AttachmentInfo? The CatalogService has a `parseParts` method that could do the parsing without the conversion.

Reply:
Refactored `AttachmentInfo` away, it does reduce the scope of change.